mantidproject/mantid
# ISIS Powder POLARIS load vanadium lifted out of loop

**Description of work.**

changed `focus.py` so that `_individual_run_focusing` loads the vanadium workspace and passes it to ` focus_one_ws` as a parameter to avoid loading the same workspace multiple times in a loop.

**To test:**
Copy the `Polaris_Test` file from `\\olympic\babylon5\Public\Sam Jenkins` edit `polaris_config.yaml` to have the correct path, then run the following code.
```
from isis_powder import
config_file_path = r"Path\to\polaris_config.yaml"
Polaris_example = Polaris(config_file=config_file_path,user_name="test")
Polaris_example.create_vanadium(first_cycle_run_no="98532")
Polaris_example.focus(input_mode="Individual", run_number="98531-98534")
```
the vanadium should only be loaded once.
Fixes #23149

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
